### PR TITLE
Add temporary patch for WebTransport

### PR DESCRIPTION
**Pull request by tidoust**
The patch rolls back the IDL to what it was before the introduction of mixins with inheritance.

### CODE REVIEW

**Comment by tidoust** [comment]
(An alternative would be for the patch to apply the changes in https://github.com/w3c/webtransport/pull/241 but the update has not been approved yet)

**Review by dontcallmedom — APPROVED** [review]
https://github.com/w3c/webtransport/pull/241 hasn't been approved, but is semantically equivalent to something that was approved before, so would probably be reasonable to pre-include. But in practice, I think reverting to the previous state is the easiest path to release this week

**Comment by foolip** [comment]
I've filed https://github.com/w3c/webtransport/issues/243 about this.

File: ed/idlpatches/webtransport.idl.patch [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ [ Exposed=(Window,Worker) ]
+-interface mixin OutgoingStream : WritableStream /* of Uint8Array */ {
++interface mixin OutgoingStream {
++  readonly attribute WritableStream writable;
+   readonly attribute Promise<StreamAbortInfo> writingAborted;
+   undefined abortWriting(optional StreamAbortInfo abortInfo = {});
+ };
+ 
+ dictionary StreamAbortInfo {
+-  [EnforceRange] octet errorCode = 0;
```
Comment (foolip) [inline ed/idlpatches/webtransport.idl.patch:28]: We don't really need to revert this change, but doing so is harmless.

**Review by foolip — APPROVED** [review]
